blog.html

```diff
@@ -0,0 +1,81 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
```

Comment:
You will need the meta tag that we referenced in week 1. Make sure it's on all of your HTML pages.